Build Error on Fedora 36 with GCC 12.

### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### Steps to reproduce the issue

#### I first cloned Open3D by:
```
git clone https://github.com/isl-org/Open3D.git
cd Open3D
```

#### Then, I build Open3D (on Fedora 36, with GCC 12.1.1) with:
```
mkdir build
cd build
cmake  ..
make -j$(nproc)
```


### Error message

```shell
I get some [-Werror = dangling-pointer =] and [-Werror = maybe-uninitialized] errors.

[-Werror=dangling-pointer=]
In file included from /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/Core:173,
                 from /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/Geometry:11,
                 from /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.h:29,
                 from /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:27:
In static member function 'static NewType Eigen::internal::cast_impl<OldType, NewType, EnableIf>::run(const OldType&) [with OldType = float; NewType = double; EnableIf = void]',
    inlined from 'NewType Eigen::internal::cast(const OldType&) [with OldType = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:464:42,
    inlined from 'const NewType Eigen::internal::scalar_cast_op<Scalar, NewType>::operator()(const Scalar&) const [with Scalar = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/UnaryFunctors.h:165:120,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_cast_op<float, double>; ArgType = const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:22,
    inlined from 'Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with BinaryOp = Eigen::internal::scalar_difference_op<double, double>; Lhs = const Eigen::Matrix<double, 3, 1>; Rhs = const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:771:69,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_abs2_op<double>; ArgType = const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:40,
    inlined from 'Eigen::internal::redux_evaluator<XprType_>::CoeffReturnType Eigen::internal::redux_evaluator<XprType_>::coeffByOuterInner(Eigen::Index, Eigen::Index) const [with XprType_ = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:383:23,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 2]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:130:34,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 1; int Length = 2]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:113:96,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 0; int Length = 3]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:113:96,
    inlined from 'static Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::Scalar Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::run(const Evaluator&, const Func&, const XprType&) [with XprType = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >; Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:224:21,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::redux(const Func&) const [with BinaryOp = Eigen::internal::scalar_sum_op<double, double>; Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:420:56,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::sum() const [with Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:465:25,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::squaredNorm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:100:46,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::norm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:112:34,
    inlined from 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:76:60:
/home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:438:12: error: using a dangling pointer to an unnamed temporary [-Werror=dangling-pointer=]
  438 |     return static_cast<NewType>(x);
      |            ^~~~~~~~~~~~~~~~~~~~~~~
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp: In static member function 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:75:43: note: unnamed temporary defined here
   75 |     auto cam_xlate = camera.GetModelMatrix().translation().cast<double>();
      |                      ~~~~~~~~~~~~~~~~~~~~~^~
In static member function 'static NewType Eigen::internal::cast_impl<OldType, NewType, EnableIf>::run(const OldType&) [with OldType = float; NewType = double; EnableIf = void]',
    inlined from 'NewType Eigen::internal::cast(const OldType&) [with OldType = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:464:42,
    inlined from 'const NewType Eigen::internal::scalar_cast_op<Scalar, NewType>::operator()(const Scalar&) const [with Scalar = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/UnaryFunctors.h:165:120,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_cast_op<float, double>; ArgType = const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:22,
    inlined from 'Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with BinaryOp = Eigen::internal::scalar_difference_op<double, double>; Lhs = const Eigen::Matrix<double, 3, 1>; Rhs = const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:771:69,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_abs2_op<double>; ArgType = const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:40,
    inlined from 'Eigen::internal::redux_evaluator<XprType_>::CoeffReturnType Eigen::internal::redux_evaluator<XprType_>::coeffByOuterInner(Eigen::Index, Eigen::Index) const [with XprType_ = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:383:23,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 1]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:130:34,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 1; int Length = 2]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:112:78,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 0; int Length = 3]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:113:96,
    inlined from 'static Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::Scalar Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::run(const Evaluator&, const Func&, const XprType&) [with XprType = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >; Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:224:21,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::redux(const Func&) const [with BinaryOp = Eigen::internal::scalar_sum_op<double, double>; Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:420:56,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::sum() const [with Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:465:25,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::squaredNorm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:100:46,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::norm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:112:34,
    inlined from 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:76:60:
/home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:438:12: error: using a dangling pointer to an unnamed temporary [-Werror=dangling-pointer=]
  438 |     return static_cast<NewType>(x);
      |            ^~~~~~~~~~~~~~~~~~~~~~~
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp: In static member function 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:75:43: note: unnamed temporary defined here
   75 |     auto cam_xlate = camera.GetModelMatrix().translation().cast<double>();
      |                      ~~~~~~~~~~~~~~~~~~~~~^~
In static member function 'static NewType Eigen::internal::cast_impl<OldType, NewType, EnableIf>::run(const OldType&) [with OldType = float; NewType = double; EnableIf = void]',
    inlined from 'NewType Eigen::internal::cast(const OldType&) [with OldType = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:464:42,
    inlined from 'const NewType Eigen::internal::scalar_cast_op<Scalar, NewType>::operator()(const Scalar&) const [with Scalar = float; NewType = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/UnaryFunctors.h:165:120,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_cast_op<float, double>; ArgType = const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:22,
    inlined from 'Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::binary_evaluator<Eigen::CwiseBinaryOp<BinaryOp, Lhs, Rhs>, Eigen::internal::IndexBased, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with BinaryOp = Eigen::internal::scalar_difference_op<double, double>; Lhs = const Eigen::Matrix<double, 3, 1>; Rhs = const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:771:69,
    inlined from 'Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::CoeffReturnType Eigen::internal::unary_evaluator<Eigen::CwiseUnaryOp<UnaryOp, ArgType>, Eigen::internal::IndexBased>::coeff(Eigen::Index, Eigen::Index) const [with UnaryOp = Eigen::internal::scalar_abs2_op<double>; ArgType = const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CoreEvaluators.h:585:40,
    inlined from 'Eigen::internal::redux_evaluator<XprType_>::CoeffReturnType Eigen::internal::redux_evaluator<XprType_>::coeffByOuterInner(Eigen::Index, Eigen::Index) const [with XprType_ = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:383:23,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, 1>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:130:34,
    inlined from 'static Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::Scalar Eigen::internal::redux_novec_unroller<Func, Evaluator, Start, Length>::run(const Evaluator&, const Func&) [with Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >; int Start = 0; int Length = 3]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:112:78,
    inlined from 'static Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::Scalar Eigen::internal::redux_impl<Func, Evaluator, 0, 2>::run(const Evaluator&, const Func&, const XprType&) [with XprType = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >; Func = Eigen::internal::scalar_sum_op<double, double>; Evaluator = Eigen::internal::redux_evaluator<Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:224:21,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::redux(const Func&) const [with BinaryOp = Eigen::internal::scalar_sum_op<double, double>; Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:420:56,
    inlined from 'typename Eigen::internal::traits<T>::Scalar Eigen::DenseBase<Derived>::sum() const [with Derived = Eigen::CwiseUnaryOp<Eigen::internal::scalar_abs2_op<double>, const Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Redux.h:465:25,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::squaredNorm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:100:46,
    inlined from 'typename Eigen::NumTraits<typename Eigen::internal::traits<T>::Scalar>::Real Eigen::MatrixBase<Derived>::norm() const [with Derived = Eigen::CwiseBinaryOp<Eigen::internal::scalar_difference_op<double, double>, const Eigen::Matrix<double, 3, 1>, const Eigen::CwiseUnaryOp<Eigen::internal::scalar_cast_op<float, double>, const Eigen::Block<Eigen::Matrix<float, 4, 4>, 3, 1, true> > >]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/Dot.h:112:34,
    inlined from 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:76:60:
/home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/MathFunctions.h:438:12: error: using a dangling pointer to an unnamed temporary [-Werror=dangling-pointer=]
  438 |     return static_cast<NewType>(x);
      |            ^~~~~~~~~~~~~~~~~~~~~~~
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp: In static member function 'static float open3d::visualization::rendering::Camera::CalcFarPlane(const open3d::visualization::rendering::Camera&, const open3d::geometry::AxisAlignedBoundingBox&)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/visualization/rendering/Camera.cpp:75:43: note: unnamed temporary defined here
   75 |     auto cam_xlate = camera.GetModelMatrix().translation().cast<double>();
      |                      ~~~~~~~~~~~~~~~~~~~~~^~
cc1plus: all warnings being treated as errors

[-Werror=maybe-uninitialized]
In file included from /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/Core:259,
                 from /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.h:29,
                 from /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:27:
In member function 'void Eigen::internal::add_assign_op<DstScalar, SrcScalar>::assignCoeff(DstScalar&, const SrcScalar&) const [with DstScalar = double; SrcScalar = double]',
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignCoeff(Eigen::Index, Eigen::Index) [with DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:655:26,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignCoeffByOuterInner(Eigen::Index, Eigen::Index) [with DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:669:16,
    inlined from 'static void Eigen::internal::copy_using_evaluator_DefaultTraversal_CompleteUnrolling<Kernel, Index, Stop>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>; int Index = 2; int Stop = 3]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:213:35,
    inlined from 'static void Eigen::internal::dense_assignment_loop<Kernel, 3, 2>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:457:92,
    inlined from 'void Eigen::internal::call_dense_assignment_loop(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:786:37,
    inlined from 'static void Eigen::internal::Assignment<DstXprType, SrcXprType, Functor, Eigen::internal::Dense2Dense, Weak>::run(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = Eigen::internal::add_assign_op<double, double>; Weak = void]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:955:31,
    inlined from 'void Eigen::internal::call_assignment_no_alias(Dst&, const Src&, const Func&) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:891:49,
    inlined from 'void Eigen::internal::call_assignment(Dst&, const Src&, const Func&, typename enable_if<(! evaluator_assume_aliasing<Src>::value), void*>::type) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:859:27,
    inlined from 'Derived& Eigen::MatrixBase<Derived>::operator+=(const Eigen::MatrixBase<OtherDerived>&) [with OtherDerived = Eigen::Matrix<double, 3, 1>; Derived = Eigen::Matrix<double, 3, 1>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CwiseBinaryOp.h:179:18,
    inlined from 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:984:43:
/home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/AssignmentFunctors.h:51:102: error: 'color.Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[2]' may be used uninitialized [-Werror=maybe-uninitialized]
   51 |   EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE void assignCoeff(DstScalar& a, const SrcScalar& b) const { a += b; }
      |                                                                                                    ~~^~~~
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp: In member function 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:970:25: note: 'color.Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[2]' was declared here
  970 |         Eigen::Vector3d color;
      |                         ^~~~~
In file included from /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/util/ConfigureVectorization.h:346,
                 from /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/Core:22:
In function '__m128d _mm_add_pd(__m128d, __m128d)',
    inlined from 'Packet Eigen::internal::padd(const Packet&, const Packet&) [with Packet = __vector(2) double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/arch/SSE/PacketMath.h:291:113,
    inlined from 'void Eigen::internal::add_assign_op<DstScalar, SrcScalar>::assignPacket(DstScalar*, const Packet&) const [with int Alignment = 0; Packet = __vector(2) double; DstScalar = double; SrcScalar = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/AssignmentFunctors.h:55:67,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignPacket(Eigen::Index, Eigen::Index) [with int StoreMode = 0; int LoadMode = 0; PacketType = __vector(2) double; DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:676:47,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignPacketByOuterInner(Eigen::Index, Eigen::Index) [with int StoreMode = 0; int LoadMode = 0; PacketType = __vector(2) double; DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:690:48,
    inlined from 'static void Eigen::internal::copy_using_evaluator_innervec_CompleteUnrolling<Kernel, Index, Stop>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>; int Index = 0; int Stop = 2]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:281:85,
    inlined from 'static void Eigen::internal::dense_assignment_loop<Kernel, 3, 2>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:456:81,
    inlined from 'void Eigen::internal::call_dense_assignment_loop(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:786:37,
    inlined from 'static void Eigen::internal::Assignment<DstXprType, SrcXprType, Functor, Eigen::internal::Dense2Dense, Weak>::run(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = Eigen::internal::add_assign_op<double, double>; Weak = void]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:955:31,
    inlined from 'void Eigen::internal::call_assignment_no_alias(Dst&, const Src&, const Func&) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:891:49,
    inlined from 'void Eigen::internal::call_assignment(Dst&, const Src&, const Func&, typename enable_if<(! evaluator_assume_aliasing<Src>::value), void*>::type) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:859:27,
    inlined from 'Derived& Eigen::MatrixBase<Derived>::operator+=(const Eigen::MatrixBase<OtherDerived>&) [with OtherDerived = Eigen::Matrix<double, 3, 1>; Derived = Eigen::Matrix<double, 3, 1>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CwiseBinaryOp.h:179:18,
    inlined from 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:984:43:
/usr/lib/gcc/x86_64-redhat-linux/12/include/emmintrin.h:248:46: error: '*(__m128d_u*)((char*)&color + offsetof(Eigen::Vector3d, Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[0]))' may be used uninitialized [-Werror=maybe-uninitialized]
  248 |   return (__m128d) ((__v2df)__A + (__v2df)__B);
      |                                              ^
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp: In member function 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:970:25: note: '*(__m128d_u*)((char*)&color + offsetof(Eigen::Vector3d, Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[0]))' was declared here
  970 |         Eigen::Vector3d color;
      |                         ^~~~~
In member function 'void Eigen::internal::add_assign_op<DstScalar, SrcScalar>::assignCoeff(DstScalar&, const SrcScalar&) const [with DstScalar = double; SrcScalar = double]',
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignCoeff(Eigen::Index, Eigen::Index) [with DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:655:26,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignCoeffByOuterInner(Eigen::Index, Eigen::Index) [with DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:669:16,
    inlined from 'static void Eigen::internal::copy_using_evaluator_DefaultTraversal_CompleteUnrolling<Kernel, Index, Stop>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>; int Index = 2; int Stop = 3]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:213:35,
    inlined from 'static void Eigen::internal::dense_assignment_loop<Kernel, 3, 2>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:457:92,
    inlined from 'void Eigen::internal::call_dense_assignment_loop(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:786:37,
    inlined from 'static void Eigen::internal::Assignment<DstXprType, SrcXprType, Functor, Eigen::internal::Dense2Dense, Weak>::run(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = Eigen::internal::add_assign_op<double, double>; Weak = void]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:955:31,
    inlined from 'void Eigen::internal::call_assignment_no_alias(Dst&, const Src&, const Func&) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:891:49,
    inlined from 'void Eigen::internal::call_assignment(Dst&, const Src&, const Func&, typename enable_if<(! evaluator_assume_aliasing<Src>::value), void*>::type) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:859:27,
    inlined from 'Derived& Eigen::MatrixBase<Derived>::operator+=(const Eigen::MatrixBase<OtherDerived>&) [with OtherDerived = Eigen::Matrix<double, 3, 1>; Derived = Eigen::Matrix<double, 3, 1>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CwiseBinaryOp.h:179:18,
    inlined from 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:981:45:
/home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/AssignmentFunctors.h:51:102: error: 'normal.Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[2]' may be used uninitialized [-Werror=maybe-uninitialized]
   51 |   EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE void assignCoeff(DstScalar& a, const SrcScalar& b) const { a += b; }
      |                                                                                                    ~~^~~~
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp: In member function 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:966:25: note: 'normal.Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[2]' was declared here
  966 |         Eigen::Vector3d normal;
      |                         ^~~~~~
In function '__m128d _mm_add_pd(__m128d, __m128d)',
    inlined from 'Packet Eigen::internal::padd(const Packet&, const Packet&) [with Packet = __vector(2) double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/arch/SSE/PacketMath.h:291:113,
    inlined from 'void Eigen::internal::add_assign_op<DstScalar, SrcScalar>::assignPacket(DstScalar*, const Packet&) const [with int Alignment = 0; Packet = __vector(2) double; DstScalar = double; SrcScalar = double]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/functors/AssignmentFunctors.h:55:67,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignPacket(Eigen::Index, Eigen::Index) [with int StoreMode = 0; int LoadMode = 0; PacketType = __vector(2) double; DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:676:47,
    inlined from 'void Eigen::internal::generic_dense_assignment_kernel<DstEvaluatorTypeT, SrcEvaluatorTypeT, Functor, Version>::assignPacketByOuterInner(Eigen::Index, Eigen::Index) [with int StoreMode = 0; int LoadMode = 0; PacketType = __vector(2) double; DstEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; SrcEvaluatorTypeT = Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >; Functor = Eigen::internal::add_assign_op<double, double>; int Version = 0]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:690:48,
    inlined from 'static void Eigen::internal::copy_using_evaluator_innervec_CompleteUnrolling<Kernel, Index, Stop>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>; int Index = 0; int Stop = 2]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:281:85,
    inlined from 'static void Eigen::internal::dense_assignment_loop<Kernel, 3, 2>::run(Kernel&) [with Kernel = Eigen::internal::generic_dense_assignment_kernel<Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::evaluator<Eigen::Matrix<double, 3, 1> >, Eigen::internal::add_assign_op<double, double>, 0>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:456:81,
    inlined from 'void Eigen::internal::call_dense_assignment_loop(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:786:37,
    inlined from 'static void Eigen::internal::Assignment<DstXprType, SrcXprType, Functor, Eigen::internal::Dense2Dense, Weak>::run(DstXprType&, const SrcXprType&, const Functor&) [with DstXprType = Eigen::Matrix<double, 3, 1>; SrcXprType = Eigen::Matrix<double, 3, 1>; Functor = Eigen::internal::add_assign_op<double, double>; Weak = void]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:955:31,
    inlined from 'void Eigen::internal::call_assignment_no_alias(Dst&, const Src&, const Func&) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:891:49,
    inlined from 'void Eigen::internal::call_assignment(Dst&, const Src&, const Func&, typename enable_if<(! evaluator_assume_aliasing<Src>::value), void*>::type) [with Dst = Eigen::Matrix<double, 3, 1>; Src = Eigen::Matrix<double, 3, 1>; Func = add_assign_op<double, double>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/AssignEvaluator.h:859:27,
    inlined from 'Derived& Eigen::MatrixBase<Derived>::operator+=(const Eigen::MatrixBase<OtherDerived>&) [with OtherDerived = Eigen::Matrix<double, 3, 1>; Derived = Eigen::Matrix<double, 3, 1>]' at /home/nnnnn/WorkSpace/open3d/build/eigen/src/ext_eigen/Eigen/src/Core/CwiseBinaryOp.h:179:18,
    inlined from 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)' at /home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:981:45:
/usr/lib/gcc/x86_64-redhat-linux/12/include/emmintrin.h:248:46: error: '*(__m128d_u*)((char*)&normal + offsetof(Eigen::Vector3d, Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[0]))' may be used uninitialized [-Werror=maybe-uninitialized]
  248 |   return (__m128d) ((__v2df)__A + (__v2df)__B);
      |                                              ^
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp: In member function 'open3d::geometry::TriangleMesh& open3d::geometry::TriangleMesh::MergeCloseVertices(double)':
/home/nnnnn/WorkSpace/open3d/Open3D/cpp/open3d/geometry/TriangleMesh.cpp:966:25: note: '*(__m128d_u*)((char*)&normal + offsetof(Eigen::Vector3d, Eigen::Matrix<double, 3, 1, 0, 3, 1>::<unnamed>.Eigen::PlainObjectBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::m_storage.Eigen::DenseStorage<double, 3, 3, 1, 0>::m_data.Eigen::internal::plain_array<double, 3, 0, 0>::array[0]))' was declared here
  966 |         Eigen::Vector3d normal;
      |                         ^~~~~~
cc1plus: all warnings being treated as errors
```


### Open3D, Python and System information

```markdown
- Operating system: Fedora 36
- Python version: 3.10.4 (main, Mar 25 2022, 00:00:00) [GCC 12.0.1 20220308 (Red Hat 12.0.1-0)]

- Open3D version: output from python: 
- System architecture: x86
- Is this a remote workstation?: no
- How did you install Open3D?: build from source
- Compiler version (if built from source): 12.1.1 20220507 (Red Hat 12.1.1-1) (GCC)
```


### Additional information

I think some errors have been detected due to the new version of GCC.

--- a/cpp/open3d/geometry/TriangleMesh.cpp
+++ b/cpp/open3d/geometry/TriangleMesh.cpp
@@ -967,7 +967,7 @@ TriangleMesh &TriangleMesh::MergeCloseVertices(double eps) {
         if (has_vertex_normals) {
             normal = vertex_normals_[vidx];
         }
-        Eigen::Vector3d color = {0, 0, 0};
+        Eigen::Vector3d color{0, 0, 0};
         if (has_vertex_colors) {
             color = vertex_colors_[vidx];
         }

--- a/cpp/open3d/geometry/TriangleMesh.cpp
+++ b/cpp/open3d/geometry/TriangleMesh.cpp
@@ -963,11 +963,11 @@ TriangleMesh &TriangleMesh::MergeCloseVertices(double eps) {
         new_vert_mapping[vidx] = new_vidx;
 
         Eigen::Vector3d vertex = vertices_[vidx];
-        Eigen::Vector3d normal;
+        Eigen::Vector3d normal{0, 0, 0};
         if (has_vertex_normals) {
             normal = vertex_normals_[vidx];
         }
-        Eigen::Vector3d color;
+        Eigen::Vector3d color = {0, 0, 0};
         if (has_vertex_colors) {
             color = vertex_colors_[vidx];
         }

--- a/cpp/open3d/geometry/TriangleMeshSubdivide.cpp
+++ b/cpp/open3d/geometry/TriangleMeshSubdivide.cpp
@@ -204,12 +204,12 @@ std::shared_ptr<TriangleMesh> TriangleMesh::SubdivideLoop(
         if (new_verts.count(edge) == 0) {
             Eigen::Vector3d new_vert =
                     old_mesh->vertices_[vidx0] + old_mesh->vertices_[vidx1];
-            Eigen::Vector3d new_normal;
+            Eigen::Vector3d new_normal{0, 0, 0};
             if (has_vert_normal) {
                 new_normal = old_mesh->vertex_normals_[vidx0] +
                              old_mesh->vertex_normals_[vidx1];
             }
-            Eigen::Vector3d new_color;
+            Eigen::Vector3d new_color{0, 0, 0};
             if (has_vert_color) {
                 new_color = old_mesh->vertex_colors_[vidx0] +
                             old_mesh->vertex_colors_[vidx1];

--- a/cpp/open3d/pipelines/integration/ScalableTSDFVolume.cpp
+++ b/cpp/open3d/pipelines/integration/ScalableTSDFVolume.cpp
@@ -120,7 +120,7 @@ std::shared_ptr<geometry::PointCloud> ScalableTSDFVolume::ExtractPointCloud() {
     auto pointcloud = std::make_shared<geometry::PointCloud>();
     double half_voxel_length = voxel_length_ * 0.5;
     float w0, w1, f0, f1;
-    Eigen::Vector3f c0, c1;
+    Eigen::Vector3f c0{0.0, 0.0, 0.0}, c1{0.0, 0.0, 0.0};
     for (const auto &unit : volume_units_) {
         if (unit.second.volume_) {
             const auto &volume0 = *unit.second.volume_;

--- a/cpp/open3d/visualization/rendering/Camera.cpp
+++ b/cpp/open3d/visualization/rendering/Camera.cpp
@@ -72,7 +72,8 @@ float Camera::CalcFarPlane(
     // The far plane needs to be the max absolute distance, not just the
     // max extent, so that axes are visible if requested.
     // See also RotationInteractorLogic::UpdateCameraFarPlane().
-    auto cam_xlate = camera.GetModelMatrix().translation().cast<double>();
+    Eigen::Vector3d cam_xlate =
+            camera.GetModelMatrix().translation().cast<double>();
     auto far1 = (scene_bounds.GetCenter() - cam_xlate).norm() * 2.5;
     auto far2 = cam_xlate.norm();
     auto model_size = 4.0 * scene_bounds.GetExtent().norm();

--- a/examples/cpp/MultipleWindows.cpp
+++ b/examples/cpp/MultipleWindows.cpp
@@ -86,7 +86,7 @@ private:
         }
 
         new_vis->ResetCameraToDefault();
-        auto center = bounds.GetCenter().cast<float>();
+        Eigen::Vector3f center = bounds.GetCenter().cast<float>();
         new_vis->SetupCamera(60, center, center + CENTER_OFFSET,
                              {0.0f, -1.0f, 0.0f});
         gui::Application::GetInstance().AddWindow(new_vis);

--- a/examples/cpp/OnlineSLAMUtil.h
+++ b/examples/cpp/OnlineSLAMUtil.h
@@ -542,7 +542,7 @@ protected:
                     geometry::AxisAlignedBoundingBox bbox(
                             Eigen::Vector3d(-5, -5, -5),
                             Eigen::Vector3d(5, 5, 5));
-                    auto center = bbox.GetCenter().cast<float>();
+                    Eigen::Vector3f center = bbox.GetCenter().cast<float>();
                     this->widget3d_->SetupCamera(60, bbox, center);
                     this->widget3d_->LookAt(center,
                                             center - Eigen::Vector3f{0, 1, 3},

--- a/examples/cpp/TICP.cpp
+++ b/examples/cpp/TICP.cpp
@@ -168,7 +168,7 @@ public:
             // Getting bounding box and center to
             // setup camera view.
             auto bbox = this->widget3d_->GetScene()->GetBoundingBox();
-            auto center = bbox.GetCenter().cast<float>();
+            Eigen::Vector3f center = bbox.GetCenter().cast<float>();
             this->widget3d_->SetupCamera(18, bbox, center);
             this->widget3d_->LookAt(center, center - Eigen::Vector3f{-10, 5, 8},
                                     {0.0f, -1.0f, 0.0f});

--- a/examples/cpp/TICPSequential.cpp
+++ b/examples/cpp/TICPSequential.cpp
@@ -190,7 +190,8 @@ private:
                 // Getting bounding box and center to setup camera view.
                 pcd_and_bbox_.bbox_ =
                         this->widget3d_->GetScene()->GetBoundingBox();
-                auto center = pcd_and_bbox_.bbox_.GetCenter().cast<float>();
+                Eigen::Vector3f center =
+                        pcd_and_bbox_.bbox_.GetCenter().cast<float>();
                 this->widget3d_->SetupCamera(verticalFoV, pcd_and_bbox_.bbox_,
                                              center);
             });
@@ -334,8 +335,9 @@ private:
                                     &pcd_and_bbox_.current_scan_, mat_);
 
                             // Setup camera.
-                            auto center = pcd_and_bbox_.bbox_.GetCenter()
-                                                  .cast<float>();
+                            Eigen::Vector3f center =
+                                    pcd_and_bbox_.bbox_.GetCenter()
+                                            .cast<float>();
                             this->widget3d_->SetupCamera(
                                     verticalFoV, pcd_and_bbox_.bbox_, center);
                         });

